transcom/mymove
# B-18312-multi-move-dashboard(MERGE-AFTER-B18310)(Merge-AFTER-B18309)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/Default.aspx?menu=StoryBoardPage)

## Summary

This PR introduces the Multi Move dashboard. When the service member logs in they are routed to the multi move dashboard. Had to comment out playwright tests in  playwright/tests/my/mymove 
 and playwright/tests/my/milmove/ppms because they depend on the Home page.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Ensure that the multiMove feature flag is switched on
2. Log into MilMove as a service member
3. You should have been routed to the Multi Move Dashboard

### Frontend



## Screenshots
![image](https://github.com/transcom/mymove/assets/136514590/43a01f55-d5df-4f79-92cd-373f1e368c7c)
